numpy/random/pcg64.pyx

```diff
@@ -91,8 +108,8 @@ cdef class PCG64(BitGenerator):
     cdef pcg64_state rng_state
     cdef pcg64_random_t pcg64_random_state
 
-    def __init__(self, seed=None):
-        BitGenerator.__init__(self, seed)
+    def __init__(self, seed_seq=None, inc=0):
```

Comment:
You had it right the first time.

Reply:
It seems then we do not use one of the advanced features of PCG64: the ability to use `inc` to create sub-streams.

Reply:
edit: use -> expose

Reply:
Correct, they should not be exposed directly to the user. https://github.com/numpy/numpy/issues/13164#issuecomment-501397679

Reply:
ok